pastis/PASTIS_Modes_E2E.py

```diff
@@ -0,0 +1,754 @@
+"""
+This module contains functions that calculates the field sensitivities for a given coronagraph design
+"""
+
+import os
+import time
+import functools
+from shutil import copy
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+from astropy.io import fits
+import astropy.units as u
+import hcipy as hc
+
+from config import CONFIG_INI
+import util_pastis as util
+from e2e_simulators.luvoir_imaging_onephot import LuvoirAPLC
+import pretty_figures as pf
+import astropy.table
```

Comment:
Please fix broken imports and remove unused ones.

Same like for the other `.py` script, how does this module differ from the notebook with the same name? I would like to avoid to have duplicate code on the repo.

Reply:
I fixed these in the notebook. This .py was just for backup.

Reply:
Gotcha - in this case, please delete the script itself.

Reply:
We wanted to delete this script

Reply:
deleted PASTIS_Modes_E2E.py